tests/foreman/cli/test_rhcloud_inventory.py

```diff
@@ -17,10 +17,60 @@
 :Upstream: No
 """
 import pytest
-
-
-@pytest.mark.stubbed
-def test_positive_inventory_generate_upload_cli():
+from broker.broker import VMBroker
+from nailgun import entities
+
+from robottelo import manifests
+from robottelo import ssh
+from robottelo.api.utils import upload_manifest
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.hosts import ContentHost
+from robottelo.rh_cloud_utils import get_local_file_data
+from robottelo.rh_cloud_utils import get_remote_report_checksum
+
+
+@pytest.fixture(scope="module")
+def organization_ak_setup(module_org):
+    with manifests.clone() as manifest:
+        upload_manifest(module_org.id, manifest.content)
+    ak = entities.ActivationKey(
+        content_view=module_org.default_content_view,
+        organization=module_org,
+        environment=entities.LifecycleEnvironment(id=module_org.library.id),
+        auto_attach=True,
+    ).create()
+    subscription = entities.Subscription(organization=module_org).search(
+        query={'search': f'name="{DEFAULT_SUBSCRIPTION_NAME}"'}
+    )[0]
+    ak.add_subscriptions(data={'quantity': 10, 'subscription_id': subscription.id})
+    return module_org, ak
+
+
+@pytest.fixture(scope="module")
+def virtual_host():
+    with VMBroker(nick='rhel7', host_classes={'host': ContentHost}) as vm:
+        yield vm
+
+
+@pytest.fixture(scope="module")
```

Comment:
Please switch these to single-quote strings here and other changed lines.

Reply:
updated the pr.